rust/src/lib/nm/nm_dbus/nm_api.rs

```diff
@@ -29,6 +29,11 @@ pub struct NmApi<'a> {
     auto_cp_refresh: bool,
 }
 
+pub struct NmVersionInfo {
+    version_encoded: u32,
+    capabilities: Vec<u32>,
```

Comment:
Can you use `bitflags!` for this? Example https://github.com/rust-netlink/netlink-packet-route/blob/main/src/link/link_flag.rs#L27

The `rust-netlink` has vetted it and since nmstate already depend on rust-netlink, we are not bringing new dependency.

This could make code easier to understand and maintain.

Reply:
The problem here is that the capabilities are exposed as a Vec<u32>, each u32 holding some flags. Then, to get the flag number 70, you need to check capabilities[2], bit 7. Bitflags would be great if we only had a single u32, but how to do it with a Vec<u32> of unknown size?

Reply:
In that case bitflag will not help here.

I am using this https://github.com/rust-netlink/wl-nl80211/blob/main/src/bytes.rs#L30 in rust-netlink.

But you do not need to use it in here, just a reference.